lib/containers/utils.pm

```diff
@@ -294,4 +294,25 @@ sub can_build_sle_base {
     return check_os_release('sles', 'ID') && $has_sle_registration;
 }
 
+sub reset_container_network_if_needed {
+    my ($current_engine) = @_;
+    my ($version, $sp, $host_distri) = get_os_release;
+    my $sp_version = "$version.$sp";
+
+    # This workaround is only needed from SLE 15-SP3 (and Leap 15.3) onwards.
+    # See https://bugzilla.suse.com/show_bug.cgi?id=1213811
+    if ($version eq "15" && $sp >= 3) {
+        my $runtime = get_required_var('CONTAINER_RUNTIME');
+        if ($host_distri =~ /sles|opensuse/ && $runtime =~ /docker/) {
+            if ($current_engine eq 'podman') {
+                # Only stop docker, if docker is active. This is also a free check if docker is present
+                systemctl("stop docker") if (script_run("systemctl is-active docker") == 0);
```

Comment:
Can't be use `systemctl()` also for the `if` like this?:
```suggestion
                systemctl("stop docker") if *systemctl("is-active docker") == 0);
```

Reply:
`systemctl` is based on `assert_script_run`, so no, unless you pass the `ignore_failure` parameter and at that point it's basically just a `script_run`.